Mathlib/LinearAlgebra/Span.lean

```diff
@@ -982,12 +981,14 @@ end AddCommMonoid
 
 section Field
 
-variable {K V} [Field K] [AddCommGroup V] [Module K V]
+variable [Field K] [AddCommGroup V] [Module K V]
 
 noncomputable section
 
 open Classical
 
+instance : Module K K := Semiring.toModule
```

Comment:
This needs a porting note; I think this is the "structure eta not used in TC" problem, so we should reference the Lean issue here too.

Reply:
done